config/default.js

```diff
@@ -19,6 +19,12 @@ var MIDlet = {
 var config = {
   main: "com/sun/midp/main/MIDletSuiteLoader",
   midletClassName: "RunTests",
+
+  // Ignored files are files that always exist and are empty, so we discard
+  // writes to them and return no data from reads.  It's useful for log files
+  // to which midlets write frequently but never read.  Specify one by setting
+  // a string pathname key to the boolean true value.
+  ignoredFiles: new Map(),
```

Comment:
Since we only need to add values and check if a value is present in *ignoredFiles*, we could use a *Set* instead of a *Map*.